src/formattedcode/output_spdx.py

```diff
@@ -63,50 +67,6 @@ def logger_debug(*args):
 Output plugins to write scan results in SPDX format.
 """
 
-_spdx_list_is_patched = False
```

Comment:
Why did you remove this?

Reply:
The `LICENSE_MAP` was dropped in https://github.com/spdx/tools-python/commit/cb6be81c9f0705120f3565989a7e7d5fc585e129, during the refactoring. The interface used for this patching is no longer available. As the spdx-tools are now dependent on https://github.com/nexB/license-expression, maybe there is another way of patching?

Reply:
@maxhbr Thanks for the clarification.... we need to find a way as ScanCode cannot rely on anything else but the licenses it knows about in a given version, i.e. the licenses must be the ones available in ScanCode toolkit not these provided by any other library (not even license-expression)